lib/order.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'text'
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Order
+  include Text
+  attr_reader :orders
+  def initialize
+    @cost = 0
+    @orders = []
+  end
+
+  def choose_dish(menu, dish, quantities)
+    order = menu.show.select { |item| item[:dish] == dish }.first
```

Comment:
Performance/Detect: Use `find` instead of `select.first`. (https://github.com/JuanitoFatas/fast-ruby#enumerabledetect-vs-enumerableselectfirst-code)